### ISSUE TITLE
Adding pagination 

### ISSUE THREAD

**Issue by MathiasCochet** [iRail/iRail#359]
Hey,

Is it possible to add real pagination to the connections call instead of adding 1 hour to the time of your first call? Because there are many cases where this will not show all the trains or show the same trains as your previous call.

**Comment by pietercolpaert** [comment]
I agree with @MathiasCochet that we should do this on the API level. The API cleans a lot of the mess and this is a nice feature we can  offer :)

**Comment by Bertware** [comment]
Internal HaCon APIs have support for this through scrB, scrF, context. This can be used to support pagination on the iRail level.
